src/vsr/replica.zig

```diff
@@ -2683,6 +2683,13 @@ pub fn ReplicaType(
                     self.replica,
                     self.view,
                 });
+                // Most of the time, passively abdicating is enough for the rest of the cluster to
+                // trigger a view change. But there is an edge case --- if "the rest of the cluster"
+                // is a single replica that believes itself to be a primary (in an older view), and
+                // that replica doesn't have locally uncommitted prepares, it will not itself try to
+                // abdicate, and it will not learn that a newer view has started. Sending an SVC
+                // here will have an effect of broadcasting our view number.
+                self.send_start_view_change();
```

Comment:
Two things I am wondering here:

- should we perhaps be sending an SVC here for reasons unrelated to this particular VOPR failure? Like, we need a quorum of SVC, and it feels like if there's a primary abdicating, it should be able to contribute to that quorum? 
- if there are _no_ other good reasons to SVC here, then sending SVC _just_ to broadcast our view feels hacky. It feels like we shuold use pings to keep cluster in sync regarding the current view. I know we moved back from that, as replicas aren't allowed to announce their view number until that view number hit the disk in the superblock, and just delaying ping would wreck clock synchronisation.

  But I am wondering... Can replicas just attach `replica.superblock.working.vsr_state.view` to ping messages? For _pings_, we _dont_ care, I think, if the view is older, as long as it is not newer?

Reply:
> For pings, we dont care, I think, if the view is older, as long as it is not newer?

Agreed; I think this is safe.

I lean towards putting more on ping, to avoid the "hacky" solution.

As a third option though, in the past we have considered folding SVC into ping (i.e. SVC is just a flag on ping, not a separate message). If we go that route, then these two solutions converge.

Reply:
On a morning walk, I realized that we actually need this SVC! Consider a Murphy's law scenario, where:

- only view-change quorum of replicas are alive
- moreover, they are connected as a star
- and the current primary is not in the center of the star
 
In this situation, we'd want for the center to become a new primary, but the center needs a quorum of SVCs, _including_ the one from the abdicating primary. But this is deeply into the SCC connectivity which we are not handling all that way today anyway.

I'll go with the ping solution --- it feels far superior as bullet-proof way to disseminate current view information across the cluster. Not sure why we didn't just code it that way back then, seems obvious in retrospect.